std/mem.zig

```diff
@@ -49,6 +41,13 @@ pub const Allocator = struct {
         return ptr;
     }
 
+    /// Alias of create
+    /// Call destroy with the result
+    pub fn construct(self: *Allocator, init: var) Error!*@typeOf(init) {
+      debug.warn("std.mem.Allocator.construct is deprecated;\n");
```

Comment:
I think we should just remove it. Zig has few users, no big codebases (`std` is the biggest) and there have already been many breaking changes over the past month :)
Besides, if `create`'s signature is changed anyways, then this is already a breaking change

Reply:
@andrewrk thoughts?

Reply:
Oops, saw this just now after I already merged it. I agree with @hejsil's logic and I removed it in a fixup commit as part of merging. 👍